### ISSUE TITLE
Plot raw green signal

### ISSUE THREAD

**Issue by deeenes** [saezlab/plugy#8]
In the QC folder, plot green fluorescence for each drug treatment, by each cycle, and averaging all cycles

Same for the orange channel.

**Comment by deeenes** [comment]
Alright, but what kind of plot? :)

**Comment by NgocHien-Du** [comment]
Boxplot please. Thank you. I think we already have the averaged-over-all-cycle plot for the orange channel. It is called 'control_fluorescence.png' in the QC folder.  But I would like to have in addition for each cycle.

**Comment by deeenes** [comment]
I created plots like `control_fluorescence.png` for all variables. From now on, these are named like `{variable}_overview.png`, hence the former `control_fluorescence.png` is now called `control_peak_median_overview.png`. Same figure for the green channel is called `readout_peak_median_overview.png`, and similar figures are available for the z-scores, for the readout:control ratio, and for the baseline corrected green signal. I changed the top (by sample) violin plot to a boxplot that shows the data both by cycle and for the whole experiment. See an example below:

![readout_peak_median_overview](https://user-images.githubusercontent.com/2679889/194113808-ce90457d-5545-47bd-939b-a5b76a774812.png)
